### ISSUE TITLE
Theme Builder /element/widget/render_content doesn't execute within Theme Builder templates alone, must be inside page's Post Content

### ISSUE THREAD

**Issue by pingram3541** [elementor/elementor#10039]
## Prerequisites
- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description
When using the front end action hook 'elementor/widget/render_content' to target an element directly inside a Theme Builder template it does't run.

However, when using it to target a widget inside a normal Elementor page builder instance that also uses a Theme Builder template through the Post Content widget, it does, but it executes twice as reported here https://github.com/elementor/elementor/issues/9952.

Here is my test code:
```
add_action( 'elementor/widget/render_content', function( $content, $widget ){
        if ( 'html' === $widget->get_name() ) {
                write_log('test'); //custom logging function
        }
}, 10, 2 );
```

## Steps to reproduce
1. Add the code above to functions.php
2a. Create a new page that does not use any Theme Builder templates, add an HTML widget, save and render on the front end. (result - a single event is logged as expected)
2b. Create a new page that uses a Theme Builder template that includes a Post Content widget, edit the page using the normal page builder, add an HTML widget, save and render the page on the front end. (result - two identical events are logged as if page builder causes duplicate)
2c. Create a new page that uses a Theme Builder template that includes an HTML element within the Theme Builder template itself, save and render the page on the front end. (result - no event is logged at all even though html element is rendered on the front end)

In the last example, let's say I have some custom post types that are automatically generated and I would like to use Elementor Theme Builder to provide the header and footer and general page design traditionally but instead of page builder content for the page content, I'd like to query some post meta for an id string, perform a loop and render that instead for the content.  Or maybe it uses post data to generate a custom shortcode to execute.

Scenario 2c is very desirable.

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

**Comment by dbjpanda** [comment]
@pingram3541 Did you find any solution?

**Comment by pingram3541** [comment]
@dbjpanda Hello, yes this issue has been resolved and is no longer present in the now current version of 3.14.1.

I tested using the following code in a child theme functions.php:

```
if ( ! function_exists('write_log')) {
   function write_log ( $log )  {
      if ( is_array( $log ) || is_object( $log ) ) {
         error_log( print_r( $log, true ) );
      } else {
         error_log( $log );
      }
   }
}
add_action( 'elementor/widget/render_content', 'o__t', 10, 2 );
function o__t( $content, $widget ){
    $settings = $widget->get_settings();
    if( 'o__t' === $settings['_element_id'] ){
        write_log('<<< hello!');
        return $content;
    } else {
        return $content;
    }
}
```

1) I then created a new page and set it to use the Elementor Canvas template so as not to use the Theme Builder's Single Page Template.

2) I add a few widgets to the page and set one of them with a custom id of o__t (some random id unlikely to be used)

3) I viewed on the front end and verified in my debug.log file I had a single instance of <<< hello! in the log.

This concludes scenario like 2a above. I cleared the log file. 

1) I then removed the Elementor Canvas template setting the page template back to default (Theme Builder Single Page template)

2) I viewed on the front end and verified in my debug.log file again I had a single instance of <<< hello! in the log.

This concludes scenario like 2b above except that it properly now results in a `single execution` vs `two`. I cleared the log file. 

1) I removed the custom id of o__t from the widget within the page and assigned it to a widget (outside the loop) in the Theme Builder's Single Page template file, a social share icon widget.

2) I viewed on the front end and verified in my debug.log file again I had a single instance of <<< hello! in the log.

This concludes scenario like 2c above except that it properly now results in a `single execution` vs `none`. I cleared the log file.
